src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star__stars"></div>
+
+      <div class="star__stars"></div>
+
+      <div class="star__stars"></div>
+
+      <div class="star__stars"></div>
+
+      <div class="star__stars"></div>
```

Comment:
star__stars are not **multiline**, so no need to separate them.
```suggestion
    <div class="stars stars--0">
      <div class="star__stars"></div>
      <div class="star__stars"></div>
      <div class="star__stars"></div>
      <div class="star__stars"></div>
      <div class="star__stars"></div>
```